Harmonize javascript 6 (#85)
kuzzleio/documentation-V2: src/sdk-reference/js/6/index/delete/index.md

description: Deletes an index
---

# delete

Deletes an entire data `<index>` from Kuzzle.

## Arguments

```javascript
delete (index, [options])
```

<br/>

| Arguments | Type   | Description      |
| --------- | ------ | ---------------- |
| `index`   | <pre>string</pre> | Index name       |
| `options` | <pre>object</pre> | Query options |

### **options**

Additional query options

| Property     | Type<br/>(default)    | Description   |
| -------------- | --------- | ------------- |
|  `queuable`  |  <pre>boolean</pre> <br/>(`true`) |  Make this request queuable or not  |

## Resolve

> Resolves to a `boolean` indicating whether the index was successfully deleted in the Elastic cluster.

Reviewer: ```suggestion
Resolves to a `true` if index was successfully deleted from ElasticSearch cluster, `false`otherwise.
```
